beacon-chain/types/block.go

```diff
@@ -178,16 +178,20 @@ func (b *Block) IsValid(
 		log.Errorf("Can not get proposer index %v", err)
 		return false
 	}
-	log.Infof("Proposer index: %v", proposerIndex)
 	if !shared.CheckBit(b.Attestations()[0].AttesterBitfield, int(proposerIndex)) {
 		log.Errorf("Can not locate proposer in the first attestation of AttestionRecord %v", err)
 		return false
 	}
 
-	for index, attestation := range b.Attestations() {
-		if !b.isAttestationValid(index, chain, aState, cState, parentSlot) {
-			log.Debugf("attestation invalid: %v", attestation)
-			return false
+	if enableAttestationValidity {
```

Comment:
Same here. We're not verifying the actual signatures, so valid attestations can be generated fairly trivially using the crystallized state. Am I missing something here?

Reply:
@terenc3t can help answer this

Reply:
It wasn't working due to indexing errors and we figured it's more clear to skip attestation verification at once because we are not verifying public keys and signatures. It could be confusing for people to think beacon node verifies attestation but not the signature. Lmk if there's more thoughts or if anyone wants to fix this. I'm ok with enabling it